### PR TITLE
Add bitcoind ZMQ configuration; Update API doc

### PR DESCRIPTION
**Pull request by ouziel-slama**

### CODE REVIEW

File: docs/advanced/api-v2/node-api.md [hunk 1]
```diff
@@ -3,14 +3,12 @@ title: API v2
...
 
 FORMAT: 1A
-HOST: https://api.counterparty.io
+HOST: https://api.counterparty.io:4000
 
 # Counterparty Core API
 
 The Counterparty Core API is the recommended (and only supported) way to query the state of a Counterparty node. 
 
-Please see [Apiary](https://counterpartycore.docs.apiary.io/) for interactive documentation.
```
Comment (adamkrellenstein) [inline docs/advanced/api-v2/node-api.md:12]: Why delete this?

File: docs/basics/manual-installation.md [hunk 2]
```diff
@@ -29,6 +29,19 @@ txindex=1
 prune=0
 mempoolfullrbf=1
 rpcworkqueue=100
+zmqpubrawtx=tcp://0.0.0.0:9332
```
Comment (adamkrellenstein) [inline docs/basics/manual-installation.md:32]: okay do we have to update Docker Compose and Terraform as well?

**Review by adamkrellenstein — APPROVED** [review]

File: docs/advanced/api-v2/node-api.md (hunk 1 above)
Comment (ouziel-slama) [reply docs/advanced/api-v2/node-api.md:12]: oops! not voluntary! will add this sentence in harcoded part!

File: docs/basics/manual-installation.md (hunk 2 above)
Comment (ouziel-slama) [reply docs/basics/manual-installation.md:32]: yep! done for docker compose!

**Pull request merged**